cmd/cri-containerd/options/options.go

```diff
@@ -109,23 +109,20 @@ type CRIConfig struct {
 	// RootDir is the root directory path for managing cri-containerd files
 	// (metadata checkpoint etc.)
 	RootDir string `toml:"root_dir" json:"rootDir,omitempty"`
-}
-
-// Config contains toml config related cri-containerd daemon.
-type Config struct {
-	CRIConfig `toml:"-"`
+	// TODO(random-liu): Remove following fields when we no longer support cri-containerd
+	// standalone mode.
 	// CgroupPath is the path for the cgroup that cri-containerd is placed in.
-	CgroupPath string `toml:"cgroup_path"`
+	CgroupPath string `toml:"cgroup_path" json:"cgroupPath,omitempty"`
```

Comment:
:-)

Reply:
Yeah, added back because they are part of `info` again with this change. :(